src/Food/food.py

```diff
@@ -1,8 +1,16 @@
+from datetime import date, timedelta
+
 class Food(object):
     def __init__(self, name, expiration_date, use_by_length):
         self.name = name
-        self.expiration_date = expiration_date
-        self.use_by_length = use_by_length
+        self.expiration_date = date.fromisoformat(expiration_date)
+        self.use_by_length = timedelta(days = use_by_length)
```

Comment:
I feel like we should add a comment or change the name of `use_by_length` to make it more clear what it is

Reply:
Done!

Reply:
Done!